src/core/game.ts

```diff
@@ -31,7 +32,7 @@ export interface GameState {
 };
 
 export class Game {
-  static settings = { minPlayers: 2, maxPlayers: 4 };
+  static settings = { minPlayers: 2, maxPlayers: 2 };
```

Comment:
That was just for testing, figured it'd be useful to keep for now since we won't need much >2 player testing for a while